rmgpy/tools/extractInfoFromckcsv.py

```diff
@@ -192,4 +194,70 @@ def getROPFlux(spc_rop_dict, species_string, rxn_index):
             if rxnNum == rxn_index:
                 flux = flux_tup[1]
                 return flux
-    return []
\ No newline at end of file
+    return []
+
+def showTheMostDominantSpecies(dominant_list, moleFraction_dict, species_identifier_dict, displayMW=False):
```

Comment:
I suggest put this block of code in the ipython notebook, just like other functionalities so that users can have the freedom to customize the printing and logic